### PR TITLE
provide classindex as osgi bundle

### PR DESCRIPTION
**Pull request by Yannic92**
Hi,
we in Eclipse Ditto would benefit from having ClassIndex as OSGI bundle.
I provided this PR to enable OSGI bundling.
Please let me know if any changes are required :)

### CODE REVIEW

**Comment by Yannic92** [comment]
@sentinelt Is there anything I need to change before this PR can be accepted?

File: classindex/pom.xml [hunk 1]
```diff
@@ -127,6 +127,18 @@
 					<testTarget>8</testTarget>
 				</configuration>
 			</plugin>
+            <plugin>
```
Comment (sentinelt) [inline classindex/pom.xml]: This file uses tab for indentation, please change spaces to tabs for consistency

File: pom.xml [hunk 2]
```diff
@@ -29,4 +29,16 @@
 		<connection>scm:git:https://github.com/atteo/classindex.git</connection>
 		<url>https://github.com/atteo/classindex</url>
 	</scm>
+
+	<build>
+		<pluginManagement>
+			<plugins>
+                <plugin>
```
Comment (sentinelt) [inline pom.xml]: This file uses tab for indentation, please change spaces to tabs for consistency

File: classindex/pom.xml (hunk 1 above)
Comment (Yannic92) [reply classindex/pom.xml]: You're right. Sorry. It's done.

File: pom.xml (hunk 2 above)
Comment (Yannic92) [reply pom.xml]: You're right. Sorry. It's done.

**Comment by thjaeckle** [comment]
@sentinelt did you consider releasing classindex with that OSGi bundle enhancement?
Would like to use it in an OSGi environment and this is the one thing missing in the latest release :)

**Comment by sentinelt** [comment]
Just released 3.7. Should be in Maven Central in a few days.

**Pull request merged**